me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/IDmeWebVerify.java

```diff
@@ -337,12 +355,12 @@ private String createRegisterAffiliationUrl(IDmeAffiliationType affiliationType)
    * Creates the URL for adding a new connection type
    *
    * @param connectionType the connection that should be added
-   * @param scope The type of group verification.
+   * @param scope          The type of group verification.
    * @return URL: with proper formatted request
    */
   private String createRegisterConnectionUrl(IDmeConnectionType connectionType, IDmeScope scope) {
     String url = idMeWebVerifyGetAuthUri
-        .replace(CLIENT_ID_KEY, clientID)
+        .replace(CLIENT_ID_KEY, clientId)
         .replace(REDIRECT_URI_KEY, redirectURI)
         .replace(RESPONSE_TYPE_KEY, "token")
```

Comment:
There are 4 types of calls, the get auth code request is the only call that should have type "token" . @mats-claassen is it right?